agent-control/src/cli/install/agent_control.rs

```diff
@@ -29,6 +29,7 @@ impl DynamicObjectListBuilder for InstallAgentControl {
     fn build_dynamic_object_list(
         &self,
         namespace: &str,
+        release_name: &str,
```

Comment:
are we making as well the agent-control-deployment name configurable? if so title and description should be updated 😄

Reply:
Which title and description? This PR is updated. `feat: make flux release name configurable`

> are we making as well the agent-control-deployment name configurable?

Next task I'll work on. I didn't want to put everything in the same one. More complexity for no benefit.

Reply:
Here where? The `InstallData` is shared between the "install agent control" and "install flux" operations of the cli. I have to do it that way, it I don't want to put a lot of effort into separating that specific field just for this PR.